src/Sylius/Bundle/AdminBundle/Resources/config/routing/shipment.yml

```diff
@@ -17,16 +17,10 @@ sylius_admin_shipment_ship:
     path: /shipments/{id}/ship
     methods: [PUT]
     defaults:
-        _controller: sylius.controller.shipment:applyStateMachineTransitionAction
```

Comment:
🚨 major bc break 🚨 all of resource functionalities are dropped with the new controller, where people may rely on them heavily.

This change has to reverted

Reply:
I've reverted these changes and added new route here